HerPublicWebsite.BusinessLogic/Services/RegularJobs/RegularJobsService.cs

```diff
@@ -45,4 +56,48 @@ public async Task RunNightlyTasksAsync()
             await dataProvider.PersistAllChangesAsync();
         }
     }
+
+    public async Task<IList<ReferralRequest>> GetReferralsPassedTenWorkingDayThresholdWithNoFollowUp()
+    {
+        var startDate = await AddWorkingDaysToDateTime(DateTime.Now, -10);
+        var newReferrals = await dataProvider.GetReferralRequestsWithNoFollowUpAfterDate(startDate);
+        return newReferrals;
+    }
+
+    public async Task<DateTime> AddWorkingDaysToDateTime(DateTime initialDateTime, int workingDaysToAdd)
```

Comment:
I've updated this comment a bit, because I realised that you won't be able to make it an extension method with the configured endpoint. It also didn't feel quite right to have an API call in an inocuous looking extension method.